transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -29,39 +29,19 @@ models:
         description: 2020 Census tract code
       - name: block
         description: 2020 Census tabulation block number
-      - name: geoid
+      - name: block_geoid
         description: >
           Census block identifier; a concatenation of 2020 Census state FIPS code, 2020
           Census county FIPS code, 2020 Census tract code, and 2020 Census block number
-      - name: name
-        description: >
-          2020 Census tabulation block name; a concatenation of 'Block'
-          and the tabulation block number
-      - name: geometry
-        description: The spatial component of geographic features
-    tests:
-      - dbt_utils.equal_rowcount:
-          compare_model: source('building_footprints', 'california_building_footprints')
```

Comment:
A shame to lose this test, but with the inner join, the number of footprints is no longer conserved! One approach to restore it would be to do the "in-California" filtering in an intermediate model, then compare the row-count with that.

Reply:
We can make that a to do!